corehq/apps/integration/kyc/models.py

```diff
@@ -188,28 +188,31 @@ def kyc_last_verified_at(self):
         return self.user_data.get('kyc_last_verified_at')
 
     @property
-    def kyc_is_verified(self):
-        value = self.user_data.get('kyc_is_verified')
+    def kyc_verification_status(self):
+        value = self.user_data.get('kyc_verification_status')
         # value can be '' when field is defined as a custom field in custom user data
-        assert value in (None, 'True', 'False', '')
-        if value == 'True':
-            return True
-        if value == 'False':
-            return False
-        return None
+        assert value in (
+            KycVerificationStatus.PENDING,
+            KycVerificationStatus.PASSED,
+            KycVerificationStatus.FAILED,
+            ''
+        )
+        if value == '':
+            return KycVerificationStatus.PENDING
+        return value
```

Comment:
Nit: This could be simplified to:
```suggestion
        return value or KycVerificationStatus.PENDING
```